spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/catalog/TestTransformTableChange.java

```diff
@@ -35,4 +36,37 @@ void testTransformRemoveProperty() {
         (com.datastrato.gravitino.rel.TableChange.RemoveProperty) tableChange;
     Assertions.assertEquals("key", gravitinoRemoveProperty.getProperty());
   }
+
+  @Test
+  void testTransformAddColumn() {
```

Comment:
could you extend the test to cover transformColumnPosition related logic, `After` `Before` and `Default`

Reply:
sure